### PR TITLE
[New Rule]  Incoming DCOM Lateral Movement with ShellBrowserWindow or ShellWindow

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/483

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
example of match, we can add in future authentication event to the sequence if needed 

image

File: rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+creation_date = "2020/11/06"
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via the ShellBrowserWindow or ShellWindows Application COM Object. This behavior may indicate an attacker abusing a DCOM application to stealthy move
+laterally.
```
Comment (threat-punter) [inline rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml:11]: Identifies use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via
the ShellBrowserWindow or ShellWindows Application COM Object. This behavior may indicate an attacker abusing a DCOM
application to stealthy move laterally.

File: rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+"""
+from = "now-9m"
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Incoming DCOM Lateral Movement with ShellBrowserWindow or ShellWindows"
+references = ["https://enigma0x3.net/2017/01/23/lateral-movement-via-dcom-round-2/"]
+risk_score = 41
```
Comment (threat-punter) [inline rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml:19]:
```suggestion
risk_score = 47
```

**Review by threat-punter — APPROVED** [review]

File: rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml [hunk 3]
```diff
@@ -0,0 +1,47 @@
+[metadata]
+creation_date = "2020/11/06"
+maturity = "production"
+updated_date = "2020/11/06"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via the ShellBrowserWindow or ShellWindows Application COM Object. This behavior may indicate an attacker abusing a DCOM application to stealthy move
+laterally.
```
Comment (bm11100) [inline rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml:10]:
```suggestion
Identifies use of Distributed Component Object Model (DCOM) to run commands from a remote host, which are launched via the ShellBrowserWindow or ShellWindows Application COM Object. This behavior may indicate an attacker abusing a DCOM application to stealthily move
laterally.
```

Comment (bm11100) [reply rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml:10]: needs linting

File: rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+name = "Incoming DCOM Lateral Movement with ShellBrowserWindow or ShellWindows"
+references = ["https://enigma0x3.net/2017/01/23/lateral-movement-via-dcom-round-2/"]
+risk_score = 47
+rule_id = "8f919d4b-a5af-47ca-a594-6be59cd924a4"
+severity = "medium"
+tags = ["Elastic", "Host", "Windows", "Threat Detection", "Lateral Movement"]
+type = "eql"
+
+query = '''
+sequence by host.id with maxspan=5s
```
Comment (bm11100) [inline rules/windows/lateral_movement_dcom_shellwindow_shellbrowserwindow.toml:25]: validated logic in master build

**Review by bm11100 — APPROVED** [review]

**Pull request merged**